app/views/pages/security.html.erb

```diff
@@ -0,0 +1,142 @@
+<% @title = "Security" %>
+
+<div class="l-colspan t-body">
+  <p>
+    Found a security issue with RubyGems or RubyGems.org?
+    Please follow these steps to report it.
+  </p>
+
+  <h2>Reporting a security issue</h2>
+
+  <p>
+    Before continuing, please ensure this is a security issue for the RubyGems
+    client or the RubyGems.org service. For all vulnerabilites with individual
+    gems, follow our guide on <a href="http://guides.rubygems.org/security/#reporting-security-vulnerabilities">
+      reporting security issues</a> with others' gems. If it's a security issue
+    with the Ruby on Rails framework, see the <a href="http://rubyonrails.org/security/">
+      Rails Security</a> guide.
+  </p>
+
+  <p>
+    <strong>
+      For any security bug or issue with the RubyGems client or
+      RubyGems.org service, please email <a href="mailto:security@rubygems.org">
+        security@rubygems.org</a> with details about the problem.
+    </strong>
+  </p>
+
+  <p>
+    <small>
+      Please note: the <a href="https://groups.google.com/forum/#!forum/rubygems-developers">
+        rubygems-developers mailing list</a>, the
+      <a href="https://groups.google.com/forum/#!forum/rubygems-org">rubygems.org
+        mailing list</a>, and the <a href="ircs://irc.freenode.net:6697/#rubygems">
+        #rubygems</a> IRC channel are public areas.
+
+      If escalating to these places, please do not discuss your issue,
+      simply say that you’re trying to get a hold of someone from the security
+      team. Thanks in advance for responsibly disclosing your security issue.
+    </small>
+  </p>
+
+  <h2>Your Security</h2>
+
+  <h3>Securely pushing Gems</h3>
+  <p>
+    You may wish to <a href="http://guides.rubygems.org/security/#building-gems">
+      sign your gems</a> before releasing them to RubyGems.org.
+  </p>
+
+  <h3>Securely downloading Gems</h3>
+  <p>
+    The SHA256hash provided with files on RubyGems.org exists only to provide some
+    download corruption protection. It is not intended to provide any sort of
+    security regarding tampering. Please use GPG signing for that.
+  </p>
+
+  <p>
+    Also, see <a href="https://gist.github.com/bf4/4223e83e8becacfb2a8e">this gist</a>.
+  </p>
```

Comment:
I presume

>  Please use GPG signing for that.

should be removed, and replaced with a link to the security guide?

I included the gist https://gist.github.com/bf4/4223e83e8becacfb2a8e in case there's anything in it we would want to include on this page or the guide. re: updating the rubygems client itself in case of security issues